src/page/utkast/skjema-section/begrunnelse/begrunnelse.tsx

```diff
@@ -1,14 +1,14 @@
-import { useEffect, useState } from 'react';
-import { BegrunnelseTipsInnhold } from './begrunnelse-tips-innhold';
+import React, { useEffect, useState } from 'react';
```

Comment:
```suggestion
import { useEffect, useState } from 'react';
```
Skal vel ikke være behov for å importere React her? Linteren klager uansett, så byggingen feiler!

Reply:
Oisann 😅 Trur det er noko i IntelliJ som importerer react når ein limar inn kode frå andre filer, og denne gongen oppdaga eg det ikkje då eg pusha.

Eg skal leggje til byggsjekk for PR-ar i dette repoet også etter kvart, så kanskje eg oppdagar linting-byggfeil litt tidlegare neste gong